aioshelly/rpc_device/wsrpc.py

```diff
@@ -50,21 +50,20 @@
 
 def _receive_json_or_raise(msg: WSMessage) -> dict[str, Any]:
     """Receive json or raise."""
+    if msg.type is WSMsgType.TEXT:
```

Comment:
Most of the times we are going to get `WSMsgType.TEXT` message so it would be better to check this first